src/html.sortable.js

```diff
@@ -685,6 +692,9 @@ var sortable = function (sortableElements, options) {
       if (!dragging || !_listsConnected(sortableElement, dragging.parentElement)) {
         return
       }
+      if (maxItems && _filter(_getChildren(sortableElement), _data(sortableElement, 'items')).length >= maxItems) {
```

Comment:
We want to know how many items the sortable contains. If we just count the children of the sortable, we'll count the placeholder and anything else that's in there at the time&mdash;we don't know how many non-"items" are inside the sortable, so no constant offset will work in all cases.

If we filter the children by the `items` option we don't need an offset, but it only works if the user specified that option in a way that excludes the placeholder and anything else inside the sortable. This could be explicitly mentioned in the documentation.

I think your solution of filtering for sortable-item-id is the right thing to do, and lacking that change having no offset (and mentioning this in the doc) is second-best.